src/components/ClosePurchaseOrderModal.vue

```diff
@@ -116,7 +116,30 @@ export default defineComponent({
           handler: async() => {
             await this.updatePOItemStatus()
             modalController.dismiss()
-            this.router.push('/purchase-orders')
+          }
+        }]
+      });
+      return alert.present();
+    },
+    async serverErrorAlert(error: any) {
+      const message = error.response?.data?.error?.message || 'Failed to update the status of purchase order items.';
+
+      const alert = await alertController.create({
+        header: translate('Error while receiving'),
+        message,
+        buttons: [{
+          text: translate('Copy & Dismiss'),
+          role: 'copyAndDismiss',
+          handler: async() => {
+            copyToClipboard(message)
+            return;
+          }
+        },
+        {
+          text: translate('Dismiss'),
+          role: 'dismiss',
```

Comment:
For dismiss buttons, as we want to have alert being dismissed automatically, we should use role as `cancel` and in that case no need for handler for that button.